orttraining/orttraining/include/runner/orttraining_c_api.h

```diff
@@ -0,0 +1,308 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) SignalPop LLC. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <string.h>
+#include <runner\training_runner.h>
```

Comment:
>er\t [](start = 14, length = 4)

Forward slash is a C/C++

Reply:
Yes.  It is part of the orttraining\models tree at https://github.com/MyCaffe/onnxruntime/tree/master/orttraining/orttraining/models/runner/training_runner.h

Reply:
Well, can a 3rd party user include it w/o MyCaffe? We are not shipping MyCaffe.

---
In reply to: [544590014](https://github.com/microsoft/onnxruntime/pull/6141#discussion_r544590014) [](ancestors = 544590014)

Reply:
the training_runner.h is already part of the orttraining/models tree - it is already provided by the OnnxRuntime in its git clone.

Reply:
The training_runner.h is not part of MyCaffe